pkg/util/util.go

```diff
@@ -172,6 +166,68 @@ func EmbeddedClusterVersion() string {
 	return os.Getenv("EMBEDDED_CLUSTER_VERSION")
 }
 
+func ReplicatedAPIEndpoint(license *kotsv1beta1.License) (string, error) {
+	if IsEmbeddedCluster() {
+		if endpoint := os.Getenv("REPLICATED_API_ENDPOINT"); endpoint != "" {
+			return maybePrependHTTPS(endpoint), nil
+		}
+		return "", errors.New("REPLICATED_API_ENDPOINT environment variable is required")
+	}
+
+	if license != nil && license.Spec.Endpoint != "" {
+		return maybePrependHTTPS(license.Spec.Endpoint), nil
+	}
+	return DefaultReplicatedAPIEndpoint(), nil
+}
+
+func maybePrependHTTPS(endpoint string) string {
+	if strings.HasPrefix(endpoint, "http://") || strings.HasPrefix(endpoint, "https://") {
+		return endpoint
+	}
+	return fmt.Sprintf("https://%s", endpoint)
+}
```

Comment:
I'm perfectly happy to always pass http/https to the chart here

Reply:
updated